.changeset/pre.json

```diff
@@ -0,0 +1,30 @@
+{
+  "mode": "pre",
+  "tag": "next",
+  "initialVersions": {
+    "@latticexyz/cli": "1.42.0",
```

Comment:
is this the version the prerelease will be prefixed with? if so we should change it to 2.0.0

Reply:
Yes this generated file will be the correct version via adding major changeset versions to the packages

Reply:
remove me